### PR TITLE
Deploy "we're hiring" banner to production!

### PR DESCRIPTION
**Pull request by cbf66**

### CODE REVIEW

**Comment by bdc34** [comment]
There is a file named `git` in the root of this project.  Please remote this.

**Review by bdc34 — APPROVED** [review]
Looks good.  Please remove `git`.

Since this has css changes the css file will need to be deployed to the CDN. Let' me know if you want me to do it for you. See https://github.com/arXiv/arxiv-browse#serving-static-files-on-s3

File: browse/templates/home/special-message.html [hunk 1]
```diff
@@ -1,20 +1,21 @@
 {#- Special messages appear in a column to the side of the website intro/news section,
   each in their own column. Always apply the style "message-special" to the column.
   You can also apply the style "dark" to override the red styles and add a message with a black background instead.
-  There should preferably be just one, and never more than two. -#}
+  There should can be just one, never more than two. -#}
```
Comment (jonathanhyoung) [inline browse/templates/home/special-message.html:4]: You should can this message in favor of one with better grammar...

**Review by jonathanhyoung — APPROVED** [review]

File: browse/templates/home/special-message.html (hunk 1 above)
Comment (cbf66) [reply browse/templates/home/special-message.html:4]: It's just a comment ...

**Pull request merged**